svelte.config.js

```diff
@@ -1,14 +1,21 @@
 import adapter from '@sveltejs/adapter-auto';
 import preprocess from 'svelte-preprocess';
+import { mdsvex } from 'mdsvex';
 
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
+  extensions: ['.svelte', '.md'],
   kit: {
     adapter: adapter(),
   },
   // Consult https://github.com/sveltejs/svelte-preprocess
   // for more information about preprocessors
-  preprocess: preprocess(),
+  preprocess: [
+    preprocess({
+      postcss: true,
+    }),
+    mdsvex({ extensions: ['.md'] }),
```

Comment:
This allows us to use `.md` within the project, and it will be processed like any other Svelte file.